### PR TITLE
[AOGM-212] Add EMCal Digitization to Data Processing layer

### PR DESCRIPTION
**Pull request by aknospe**
Added DigitizerSpec and DigitWriterSpec (based on the corresponding code for TOF). Integrated these into the CMake file. Added EMCal to Digitizer workflow.

### CODE REVIEW

File: cmake/O2Dependencies.cmake [hunk 1]
```diff
@@ -272,15 +272,6 @@ o2_define_bucket(
...
 )
 
-o2_define_bucket(
-    NAME
-    O2FrameworkCore_benchmark_bucket
-
-    DEPENDENCIES
-    O2FrameworkCore_bucket
-    $<IF:$<BOOL:${benchmark_FOUND}>,benchmark::benchmark,$<0:"">>
-)
```
Comment (sawenzel) [inline cmake/O2Dependencies.cmake]: why did you remove this? Seems not related to this PR and the compilation seems to fails due to this

Comment (aknospe) [reply cmake/O2Dependencies.cmake]: My mistake. I have fixed it. (Change not yet committed.)

**Comment by aknospe** [comment]
@sawenzel, I fixed the pull request to solve the issue identified last week. The new commit and the original one have been squashed.

**Comment by sawenzel** [comment]
@aknospe : Could you still fix the 2 tiny clang-format errors (listed under the travis link) please?

**Comment by aknospe** [comment]
Sure. Just to be sure I understand, these are the changes proposed at lines 541-542 and lines 550-551, right?

**Comment by sawenzel** [comment]
@aknospe : Yes right!

**Comment by sawenzel** [comment]
Any news here?

**Pull request merged**